crates/bevy_window/src/event.rs

```diff
@@ -58,10 +58,18 @@ pub struct WindowCloseRequested {
 pub struct WindowClosed {
     pub id: WindowId,
 }
-/// An event that is sent whenenver the user's cursor moves.
+/// An event reporting that the mouse cursor has moved on a window.
+///
+/// The event is sent only if the cursor is over one of the application's windows.
+/// It is the translated version of [`WindowEvent::CursorMoved`] from the `winit` crate.
+///
+/// [`WindowEvent::CursorMoved`]: https://docs.rs/winit/latest/winit/event/enum.WindowEvent.html#variant.CursorMoved
 #[derive(Debug, Clone)]
 pub struct CursorMoved {
+    /// The identifier of the window the cursor has moved on.
     pub id: WindowId,
+
+    /// The position of the cursor, in window coordinates.
     pub position: Vec2,
```

Comment:
Not sure about the accepted style on adding a whitespace or not here. Rustfmt seems to like both.

Reply:
No strong feelings. I'm fine to leave it like this.